profile-service/pom.xml

```diff
@@ -242,8 +242,9 @@
       <version>${javax.servlet.version}</version>
     </dependency>
     <dependency>
-      <groupId>mysql</groupId>
-      <artifactId>mysql-connector-java</artifactId>
+      <groupId>com.mysql</groupId>
+      <artifactId>mysql-connector-j</artifactId>
+      <version>8.3.0</version>
```

Comment:
Just wondering, if [this page](https://mvnrepository.com/artifact/mysql/mysql-connector-java/8.0.33) is relevant here or not

Reply:
There is a notice on that page indicates the dependency is now published under the group and artifact I have used.